Python/3-dict.py

```diff
@@ -0,0 +1,26 @@
+class SingletoneDict:
```

Comment:
`class SingletoneDict` is redundant
_state_dict can be stored directly in Singletone class
```python
class Singletone:
    _state_dict = {}

    def __init__(self, arg):
        self.__dict__ = self._state_dict
        self.val = arg

    def __str__(self):
        return self.val
```